recipes/boost_histogram/conda_build_config.yaml

```diff
@@ -0,0 +1,12 @@
+c_compiler:                    # [win]
+  - vs2017                     # [win]
+  - vs2017                     # [win]
+cxx_compiler:                  # [win]
+  - vs2017                     # [win]
+  - vs2017                     # [win]
+vc:                            # [win]
+  - 14.1                       # [win]
+  - 14.1                       # [win]
+python:                        # [win]
+  - 3.6                        # [win]
+  - 3.7                        # [win]
```

Comment:
Why not use selectors here?

Reply:
As it needs vs2017 or newer to be able to compile and this is the only way I know to set it up on windows. Is there a better way?

Reply:
https://github.com/conda-forge/llvmdev-feedstock/blob/192ce2e2c638672b715b92e19f2c14d47844b4b3/recipe/meta.yaml#L28-L29

Reply:
@isuruf Thanks. Do you know if it's possible to avoid this conflict with Python 2.7?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=84032&view=logs

Reply:
https://github.com/conda-forge/llvmdev-feedstock/blob/192ce2e2c638672b715b92e19f2c14d47844b4b3/recipe/meta.yaml#L23

Reply:
That seems to just skip Python 2.7 instead of allowing it to be built?

Reply:
Marking resolved as this file has been dropped.